common/app/renderers/DotcomRenderingService.scala

```diff
@@ -79,6 +79,12 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
           // if DCR returns a 400 it's because *we* failed, so frontend should return a 500
           NoCache(play.api.mvc.Results.InternalServerError("Remote renderer validation error (400)"))
             .withHeaders("X-GU-Dotcomponents" -> "true")
+        case 415 =>
+          // if DCR returns a 415 it's because we can't render a specific component, so page is not available
+          NoCache(
```

Comment:
There is a `NotFound` default cache value we could use here: https://github.com/guardian/frontend/blob/main/common/app/model/Cached.scala#L26.

In any case, I think we need to have some caching on these.

Reply:
Sorry, I forgot about the surrogate header.  You should set this header so we do for other 404s.